packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/MakeOffer/index.tsx

```diff
@@ -359,6 +360,7 @@ const MakeOffer: React.FC<Props> = (props) => {
                       addDays(new Date(), parseInt(formValues.expirationDays))
                     ),
                     offerFees,
+                    sellOrder: orderFlow.sellOrder || undefined,
```

Comment:
do you need || undefined?

Reply:
Yeah because for make offer it's not sending a sell order up

Reply:
wouldnt orderFlow.sellOrder be undefined then?